plugin.video.uzg/resources/language/resource.language.en_gb/strings.po

```diff
@@ -0,0 +1,19 @@
+# Kodi Media Center language file
```

Comment:
The correct path for Gotham is ` resources/language/English/strings.po`. `resource.language.en_gb` is only for Helix and newer.

Reply:
Sorry, I just wanted to repair subtitles. But also had to implement these changes suddenly.

Reply:
Can you remove this file and `resources/language/resource.language.nl_nl/strings.po` as `resources/language/English/strings.po` and `resources/language/Dutch/strings.po` still work for newer Kodi versions.

Reply:
Done.

Reply:
Not sure what you have done, but files are still present.
As this PR has been already merged no further action needed, as it doesn't break anything.

Reply:
Sorry, my GUI did not like that files were added and removed. He did not want tore-write history this away with a force commit.
I saw this just too late and quickly adjusted it with command line. But the PR had already happened then.

https://github.com/Opvolger/plugin.video.uzg/tree/Gotham

If there is another adjustment on the Gotham version, I will send the change